lib/diagnostic.rb

```diff
@@ -105,15 +124,25 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+  include Carnivorous
+  def roar
+  puts "ROAR!"
+  end
+end
+
+
+
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+# You would use composition when you want you want to add a behavior that may
+# apply to very distinct classes.  This would be helpful in a case where you want
+# to add behavior where inheritance doesn't make sense like a person and cat eating.
```

Comment:
In direct inheritance, you're limited to inheriting from a single source. In composition, you can draw from as many modules as you like.